composer.json

```diff
@@ -51,7 +51,7 @@
         "infection/extension-installer": "^0.1.0",
         "infection/include-interceptor": "^0.2.4",
         "justinrainbow/json-schema": "^5.2",
-        "nikic/php-parser": "^4.9",
+        "nikic/php-parser": "^4.9 <4.10",
```

Comment:
Newer version has this issue:
https://travis-ci.com/github/infection/infection/jobs/391611781#L1088

And I don't think this is the PR to fix it. It might not even be our issue.